src/audio/base_fw.c

```diff
@@ -111,6 +111,12 @@ static int basefw_config(uint32_t *data_offset, char *data)
 	sche_cfg.sys_tick_source = SOF_SCHEDULE_LL_TIMER;
 	tlv_value_set(tuple, IPC4_SCHEDULER_CONFIGURATION, sizeof(sche_cfg), &sche_cfg);
 
+	tuple = tlv_next(tuple);
+	if (IS_ENABLED(CONFIG_ADSP_IMR_CONTEXT_SAVE))
+		tlv_value_uint32_set(tuple, IPC4_FW_CONTEXT_SAVE, 1);
+	else
+		tlv_value_uint32_set(tuple, IPC4_FW_CONTEXT_SAVE, 0);
```

Comment:
I think it should work to just use `tlv_value_uint32_set(tuple, IPC4_FW_CONTEXT_SAVE, IS_ENABLED(CONFIG_ADSP_IMR_CONTEXT_SAVE))`

Reply:
is IS_ENABLED work ?